extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufExtensionsModule.java

```diff
@@ -45,5 +46,6 @@
   @Override
   public void configure(Binder binder)
   {
+    JsonConfigProvider.bind(binder, "druid.opencensus.protobuf", OpenCensusProtobufConfig.class);
```

Comment:
Is `protobuf` necessary here?  Use of OpenCensus kind of implies protobuf.  And even if it didn't wouldn't processing of resource labels be format-agnostic (protobuf or json)?

Reply:
Done.